site/content/blog/2023-04-01-zap-updates-march-2023/index.md

```diff
@@ -0,0 +1,220 @@
+The GraphQL IDE will be exposed via the ZAP API and all requests made using the IDE will be proxied through ZAP.
+
+**NodeJS API Client**  
+The [ZAP API NodeJS client](https://github.com/zaproxy/zap-api-nodejs/) and tooling around it were updated in March. The client is pending a release.
+
+**2.13.0 Release**
+We may do a major release of ZAP as soon as April 2023.
+
+
+
+## Website Updates
+The following new pages were added:
+- Blog post: [How Should We Fund Zap Development?](/blog/2023-03-09-how-should-we-fund-zap-development/)
+- [Become a Sponsor](/sponsor/)
+
+The [roadmap](/docs/roadmap/) was updated to reflect the latest status of ongoing ZAP development.
+
+
+
+## Released add-ons - Full Changelog
+In March 2023, we released updated versions of 23 add-ons:
+
+##### Community Scripts (v16)
+Added
+- httpsender/UpgradeHttp1To2.js - changes all HTTP/1.1 requests to use HTTP/2
+- standalone/devTools.js - Tools used to explore objects returned by the Java engine and better plug Nashorn objects into it
+
+Changed
+- encode-decode/double-spacer.js - adapted to the functionality of Encoder 1.0.0.
+
+Removed
+- standalone/Run report.js - no longer working, the old/deprecated class that it used was removed.
+
+Fixed
+- active/User defined attacks.js - correctly escape dot character in some evidence strings.
+- targeted/curl_command_generator.js - prevent and warn on local file inclusion when generating the command.
+  Thanks to James Kettle (@albinowax) for reporting.
+
+##### Script Console (v36, v37, v38)
+Added
+- Support for inline scripts in the Automation Framework.
+
+Fixed
+- extender/Copy as curl command menu.js - prevent and warn on local file inclusion when generating the command.
+  Thanks to James Kettle (@albinowax) for reporting.
+- Do not require a name if file provided when adding script with automation job.
+
+Changed
+- Maintenance changes.
+
+##### Requester (v7.2.0)
+Fixed
+- Allow to read HTTP/2 trailing headers.
+
+Changed
+- Maintenance changes.
+
+##### Automation Framework (v0.25.1, v0.26.0)
+Added
+- Support for dynamically added browser based authentication.
+
+Fixed
+- NPE when accessing active scan job.
+
+##### Ajax Spider (v23.13.0)
+Added
+- Automation Framework - HTML elements to click support
+
+Fixed
+- Close the AJAX Spider dialogue when uninstalling the add-on.
+
+##### Quick Start (v37)
+Fixed
+- Show correct error message when unable to access the provided URL, also, add the scheme if none provided.
+- Ensure the add-on is not in use before uninstalling.
+
+##### OAST Support (v0.15.0)
+Added 
+- A context menu to paste payloads from all the supported OAST services (Issue 7665).
+
+##### Encoder (v1.1.0)
+Added
+- A context menu active on the output text areas facilitating the replacement of input text.
+- PowerShell encoder (hat tip to hackvertor/hackvertor#71 for the idea and details).
+
+##### Browser View (v6)
+Added
+- Add info and repo URLs.
+
+Changed
+- Update minimum ZAP (vto 2.12.0.)
```

Comment:
Typos?

Reply:
Yeah, I had done a find-and-replace and this got caught by accident.

Reply:
Been there 😉